src/Core/TimerFactory/TimerFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Chronos.Timer.Core
+{
+    public class TimerFactory : ITimerFactory
```

Comment:
We should talk about comments on classes - I prefer having full `<summary>` comments everywhere (as in the Interfaces), but am open to shorter comments referencing the Interface or otherwise.